packages/google-vertex/src/google-vertex-image-model.test.ts

```diff
@@ -304,5 +313,31 @@ describe('GoogleVertexImageModel', () => {
         },
       });
     });
+
+    it('should return image meta data', async () => {
+      prepareJsonResponse();
+
+      const result = await model.doGenerate({
+        prompt,
+        n: 2,
+        size: undefined,
+        aspectRatio: undefined,
+        seed: undefined,
+        providerOptions: {},
+      });
+
+      expect(result.providerMetadata?.vertex).toStrictEqual({
+        images: [
+          {
+            contentType: 'image/png',
```

Comment:
is the content type already part of the standardized response? if yes, then we do not need to duplicate it  here. if no, should it be?

Reply:
Good catch, I missed that it already is: https://github.com/vercel/ai/blob/abf9a792c25f408bafc7e0c277ceafda5ac69b35/packages/ai/core/generate-text/generated-file.ts#L20-L25

I'll update the PR

Reply:
resolved via 2ee6b2c96